uxarray/utils/numba.py

```diff
@@ -0,0 +1,49 @@
+import uxarray.constants
+
+
+def enable_jit_cache():
```

Comment:
@aaronzedwick 

I've moved these under `uxarray.utils.numba`, but they can be accessed directly under the `uxarray.utils` module. Wanted to see if you were good with this change!